packages/calypso-e2e/src/rest-api-client.ts

```diff
@@ -22,7 +26,8 @@ import type { BodyInit, HeadersInit, RequestInit } from 'node-fetch';
 /**
  * Specifies the version of WordPress.com REST API.
  */
-type EndpointVersions = '1' | '1.1' | '1.2' | '1.3';
+type EndpointVersions = '1' | '1.1' | '1.2' | '1.3' | '2';
+type EndpointNamespace = 'rest' | 'wpcom';
```

Comment:
The [WordPress.com Developer Console](https://developer.wordpress.com/docs/api/console/) only supports versions 1 through 1.3 on the front end, but hidden away in the backend code is also version 2.

Combined with the `wpcom` namespace we are able to query undocumented endpoint that can perform tasks that the `rest` endpoint cannot.

Reply:
Whoa. Thanks for this comment. This is really good to know.